opnode/withdrawals/utils.go

```diff
@@ -0,0 +1,267 @@
+package withdrawals
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+	"time"
+
+	"github.com/ethereum-optimism/optimistic-specs/l2os/bindings/l2oo"
+	"github.com/ethereum-optimism/optimistic-specs/opnode/contracts/deposit"
+	"github.com/ethereum-optimism/optimistic-specs/opnode/contracts/withdrawer"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/ethclient/gethclient"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+var L2WithdrawalAddr = common.HexToAddress("0x4200000000000000000000000000000000000016")
```

Comment:
This is no longer correct, the predeploy is at `address(42...0)`

Reply:
No zeroes?

Reply:
Also, this has been removed and I'm using the address in the `predeploys` package. FYI this address is still found in the typesecript helpers

Reply:
This is working in the tests because we init the code at `predeploy.WithdrawalsAddress` and then reference it at that address later.